packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -273,8 +272,8 @@ class BottomSheetCell extends Component {
 				return accessibilityLabel || label;
 			}
 
-			if ( isEmpty( value ) ) {
-				return isEmpty( help )
+			if ( ! value ) {
```

Comment:
As you can see below, `value` is used as a string, therefore we can safely use it as a string in the `if()`, simplifying the condition.